File: python/dse_insert.py
#!/usr/bin/env python3

# this is a insert statement in python
from dse.cluster import Cluster, ExecutionProfile, EXEC_PROFILE_DEFAULT
from dse.query import tuple_factory
from ip_address import connection

cluster = Cluster([connection.ip_address])
# session = cluster.connect()

# print session.execute("INSERT INTO crud.users (first_name,last_name,age) VALUES (BOB,BOBERSON,21)")[0]

File: nodejs/dse_insert.js
const connection = require('./ip_address')

// this is a insert statement in nodejs
connection.client.execute(
    'INSERT INTO crud.users (first_name,last_name,age) VALUES (?,?,?)',
    ['Bob','Boberson',21],
)
.then(result => console.log(result))
.catch(error => console.log(error.message));